### PR TITLE
Merge Chrultrabook #102 PR into vuepress rebase

### PR DESCRIPTION
**Pull request by CoolPenguin27**
This pr merges what I did in this [pr](https://github.com/chrultrabook/docs/pull/102) into here. This also restores the original LICENSE and README files and moves the chrultrabook controller ad into post-install.md

### CODE REVIEW

File: src/docs/recovery-mode.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+<img src="https://lh3.googleusercontent.com/Y97tvM9PaKrYPwPI3Y0QxO_t8zKj3xl9vxZbL_Xby6L3wsMhqU3UxseELrhl8wcpjtE=w500">
+
+When you do, use something like a paper clip to hold the button down, then turn on the device. This should bring your system into recovery mode.
+You can stop holding down the button at this point.
+
+**Tablets**
+
+Hold down `Volume Up` + `Volume Down` + `Power` for 10 seconds, then release. This should bring your system into recovery mode.
+
+<img src="https://kmpic.asus.com/images/2022/11/17/6bec2799-6ad7-43d5-82a3-f57bfeb795b6.png" width=500x500>
```
Comment (ethanaobrien) [inline src/docs/recovery-mode.md:39]: Can we download any external images and add them to the images directory, similar to how the coolstar paid driver pages are done

File: src/docs/developer-mode.md [hunk 2]
```diff
@@ -6,14 +6,33 @@ Enabling [developer mode](https://chromium.googlesource.com/chromiumos/docs/+/HE
...
+2. Press `CTRL` +  `D` at the same time. This should bring up a warning asking for confirmation for either "Turn OS Verification OFF" or "Enable Developer Mode".
+
+3. Press `Enter`. The system should reboot and bring you to the "You are in Developer Mode" or "OS Verification is OFF" screen.
+
+4. Press `CTRL` +  `D` to boot from internal disk.
+
+4. Setup the system like normal, but **do not enable debugging features**.
 
-Congratulations! You've enabled developer mode!
+<table>
```
Comment (ethanaobrien) [inline src/docs/developer-mode.md:29]: Can we convert this table to markdown?

**Review by ethanaobrien — CHANGES_REQUESTED** [review]

File: src/docs/developer-mode.md (hunk 2 above)
Comment (CoolPenguin27) [reply src/docs/developer-mode.md:29]: I dont know.

Comment (ethanaobrien) [reply src/docs/developer-mode.md:29]: I'll do this later then, don't worry about it

File: src/docs/recovery-mode.md (hunk 1 above)
Comment (CoolPenguin27) [reply src/docs/recovery-mode.md:39]: we also need to update the links of the current images so that they dont break

Comment (CoolPenguin27) [reply src/docs/recovery-mode.md]: probably when the vuepress rebase pr is about to be merged

**Pull request merged**